handbook/product/design/reviewing_design/index.md

```diff
@@ -13,3 +13,11 @@ As a design team, it’s our responsibility [to communicate expectations clearly
 As a teammate outside the design team, it’s your responsibility to understand [when and how your feedback will be most valuable](../../../communication/seeking-and-giving-feedback.md#giving-feedback).
 
 We have a [set of guidelines for how to seek and give great feedback](../../../communication/seeking-and-giving-feedback.md).
+
+## Reviews in Figma
+
+To make reviewing designs easier, we use the following tools in Figma:
+
+- [**Labels:**](https://www.figma.com/file/8qNcDzOXLj1hcOM76WDPN9/🛠Project-Tools?node-id=1502%3A6598) to indicate the state of the **mockups**: draft, ready for review, current (approved, ready for dev), deprecated (usually left there for historical context), conceptual (exploratory, an alternative solution) or **copy**: draft copy, current copy (approved, the most recent version).
```

Comment:
```suggestion
- [**Labels:**](https://www.figma.com/file/8qNcDzOXLj1hcOM76WDPN9/🛠Project-Tools?node-id=1502%3A6598) to indicate the state of the **design**: draft, ready for review, current (approved, ready for dev), deprecated (usually left there for historical context), conceptual (exploratory, an alternative solution) or **copy**: draft copy, current copy (approved, the most recent version).
```

Reply:
@AlicjaSuska what does this mean?

> or **copy**: draft copy, current copy (approved, the most recent version)

Are you referring to copy of a Figma file or copywriting? I'm unaware of how these would be or are used in our current workflows.

Reply:
I think @quinnkeast added those labels. In my understanding, in copy-heavy designs, we should distinguish between final and in-progress/example copy, if needed. @quinnkeast could you please confirm?